.github/workflows/deploy.yml

```diff
@@ -36,15 +36,14 @@ jobs:
             EOF
         - name: Setup
           run: |
-            # sudo apt update
-            # sudo apt install libgif-dev
+            sudo apt update
+            sudo apt install pkg-config libcairo2-dev libpango1.0-dev libssl-dev libjpeg62-dev libgif-dev
             TRAVIS_BRANCH="${{github.ref_name}}" travis/setup-git.sh # Configure git user info
             travis/install-assets.sh
```

Comment:
Travis? 🤔

Reply:
Ha! Yeah... upstream still uses Travis, so this should (in theory) make it easier to merge from them in the future.